src/test/java/org/cryptomator/cryptofs/CryptoFileChannelWriteReadIntegrationTest.java

```diff
@@ -556,6 +557,21 @@ public void testClosingChannelOfDeletedFileDoesNotThrow() {
 			});
 			Assertions.assertTrue(Files.notExists(file));
 		}
+
+		//https://github.com/cryptomator/cryptofs/issues/170
+		@Test
+		public void testWriteThenDeleteThenRead() throws IOException {
+			var bufToWrite = ByteBuffer.wrap("delete me".getBytes(StandardCharsets.UTF_8));

```

Comment:
```suggestion
			var bufToWrite = StandardCharsets.UTF_8.encode("delete me");

```

Reply:
Fixed in https://github.com/cryptomator/cryptofs/pull/172/commits/2a22af17abc35d271992836146964f9939c25d9b